doc/man7/EVP_KDF-SNMPKDF.pod

```diff
@@ -0,0 +1,112 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF-SNMPKDF - The SNMPKDF EVP_KDF implementation
+
+=head1 DESCRIPTION
+
+Support for computing the B<SNMPKDF> KDF through the B<EVP_KDF> API.
+
+The EVP_KDF-SNMPKDF algorithm implements the SNMPKDF key derivation function.
+It is defined in RFC 3414, appendix A.2.2 and is used by SNMP to derive
+encryption keys.
+Three inputs are required to perform key derivation: The hashing function
+(for example SHA1), the engine ID, and the password
+
+=head2 Identity
+
+"SNMPKDF" is the name for this implementation; it
+can be used with the EVP_KDF_fetch() function.
+
+=head2 Supported parameters
+
+The supported parameters are:
+
+=over 4
+
+=item "properties" (B<OSSL_KDF_PARAM_PROPERTIES>) <UTF8 string>
+
+=item "digest" (B<OSSL_KDF_PARAM_DIGEST>) <UTF8 string>
+
+=item "pass" (B<OSSL_KDF_PARAM_PASSWORD>) <octet string>
+
+These parameters works as described in L<EVP_KDF(3)/PARAMETERS>.
+
+=item "eid" (B<OSSL_KDF_PARAM_SNMPKDF_EID>) <octet string>
+
+This parameters set the respective values for the KDF.
```

Comment:
Try this instead: _This parameter sets the snmpEngineID value for the KDF._

Reply:
done